source/beamline/CLS/CLSQE65000Detector.cpp

```diff
@@ -158,12 +158,18 @@ bool CLSQE65000Detector::initializeImplementation(){
 }
 
 bool CLSQE65000Detector::acquireImplementation(AMDetectorDefinitions::ReadMode readMode){
-	if(!isConnected() || readMode != AMDetectorDefinitions::SingleRead)
+	if(!isConnected() || readMode != AMDetectorDefinitions::SingleRead) {
+
+		if(readMode != AMDetectorDefinitions::SingleRead) {
+		}
 		return false;
+	}
 
 	AMControl::FailureExplanation failureExplanation = startAcquisitionControl_->move(1);
-	if(failureExplanation != AMControl::NoFailure)
+	if(failureExplanation == AMControl::NoFailure) {
```

Comment:
Until 3e31c42c2fee83d181a425dc770faf9f9a9a7db3 the AMDetectorTriggerAction didn't pay attention to the return result of the call to the detector's acquire() function. The action succeeded either way. So while this function was returning the wrong value nothing was paying attention to it.